src/sentry/api/serializers/rest_framework/notification_action.py

```diff
@@ -0,0 +1,103 @@
+from typing import Dict, List, Tuple, TypedDict
+
+from rest_framework import serializers
+
+from sentry.api.serializers.rest_framework.base import CamelSnakeModelSerializer
+from sentry.api.serializers.rest_framework.project import ProjectField
+from sentry.models.notificationaction import ActionService, ActionTarget, NotificationAction
+from sentry.models.project import Project
+from sentry.services.hybrid_cloud.integration import integration_service
+
+
+def format_choices_text(choices: List[Tuple[int, str]]):
+    return ", ".join([f"'{display_text}'" for (_, display_text) in choices])
```

Comment:
You could use `oxfordize_list([display_text for _, display_text in choices])` from sentry.utils.strings. I don't know if that buys much though.